dali/sasha/saserver.cpp

```diff
@@ -360,7 +360,7 @@ int main(int argc, const char* argv[])
     #endif
         DBGLOG("Build %s", hpccBuildInfo.buildTag);
 
-        unsigned short port = serverConfig->getPropInt("@port");
+        unsigned short port = serverConfig->getPropInt("service/@port", serverConfig->getPropInt("@port"));
```

Comment:
doesn't the schema prevent "@port" being valid, and so is there any point in defaulting up legacy spec. ?

Reply:
Depends on what bare-metal looks like...